frontend/src/components/Menu.vue

```diff
@@ -6,23 +6,70 @@
-  <div @click.stop="hide" class="dimmed no-scroll">
-    <div class="vertical-container">
-      <RouterLink to="/about" class="link">
-        <label>About</label>
-      </RouterLink>
-      <!-- <RouterLink to="/" class="link">
-        <label>Events</label>
-      </RouterLink> -->
-      <RouterLink to="/resources" class="link">
-        <label>CSE Resources</label>
-      </RouterLink>
-      <RouterLink to="/sponsors" class="link">
-        <label>Sponsors</label>
-      </RouterLink>
-      <RouterLink to="/engage" class="link">
-        <label>Engage</label>
+  <div @click.stop="hide" class="cover">
+    <div class="frame">
+      <!-- Logo -->
+      <RouterLink to="/" v-on:click.native="showMenu = false">
+        <div class="fill-height" style="width: 140px; margin-left: 5px;">
+          <v-img src="@/assets/csesoc-logo-white.svg" />
+        </div>
       </RouterLink>
+      <div class="spacing-md" />
+      <div class="split">
+        <!-- Page Links -->
+        <div class="page-links-container">
+          <RouterLink to="/about" class="link">
+            <h2>001 | About</h2>
+          </RouterLink>
+          <!-- <RouterLink to="/" class="link">
+            <h2>Events</h2>
+          </RouterLink> -->
+          <RouterLink to="/resources" class="link">
+            <h2>010 | Resources</h2>
+          </RouterLink>
+          <RouterLink to="/sponsors" class="link">
+            <h2>011 | Sponsors</h2>
+          </RouterLink>
+          <RouterLink to="/engage" class="link">
+            <h2>100 | Engage</h2>
+          </RouterLink>
+        </div>
+        <!-- Social Links -->
+        <div class="social-links-container">
+          <div class="push-down">
+            <a class="link" href="https://www.facebook.com/csesoc/">
+              <p>Facebook</p>
+            </a>
+            <a class="link" href="https://www.instagram.com/csesoc_unsw/">
+              <p>Instagram</p>
+            </a>
+            <a class="link" href="https://forms.office.com/Pages/ResponsePage.aspx?id=pM_2PxXn20i44Qhnufn7o6ecLZTBorREjnXuTY-JfmBUMEpOMFBDTU1UWkhBWllWRTNPOVJFMUNCRi4u">
+              <p>Discord Community</p>
+            </a>
+            <a class="link" href="https://csesoc-community.slack.com/">
+              <p>Slack Community</p>
+            </a>
+            <a class="link" href="https://www.youtube.com/channel/UC1JHpRrf9j5IKluzXhprUJg">
+              <p>Youtube</p>
+            </a>
+            <a class="link" href="https://www.linkedin.com/company/csesoc/">
+              <p>LinkedIn</p>
+            </a>
+            <a class="link" href="#">
+              <p>TikTok</p>
+            </a>
+          </div>
+        </div>
+      </div>
+      <!-- Footer -->
+      <div class="spacing-xs" />
+      <div>
+        <p style="display: inline;">
+          <img height="14px" src="@/assets/moon-icon.png" style="display: inline; padding-right: 5px;">
+          Only Dark mode allowed here at CSESoc. Our apologies.
```

Comment:
This isn't displayed.

Reply:
I can see this in the screenshot, it's just the colour that is very hard to see.